src/Common/tests/gtest_helper_functions.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "Poco/DOM/Document.h"
+#include "Poco/DOM/NodeList.h"
+#include "Poco/DOM/NamedNodeMap.h"
+
+
+inline char *getTempFileWithContents(const char *fileContents)
+{
+    auto *file_pointer = std::tmpnam (nullptr);
+    auto *file_stream = std::fopen(file_pointer, "w+");
+    std::fputs(fileContents, file_stream);
+    std::fclose(file_stream);
+
+    return file_pointer;
+}
+
+inline std::string xmlNodeAsString(Poco::XML::Node* &pNode)
```

Comment:
Reference to pointer?

Reply:
Had to look it up and found some example I (partially) copy-pasted and had to further modify to work correctly.
But I'll change the parameter.